### PR TITLE
start date filters events

### PR DESCRIPTION
**Pull request by feedmypixel**
- Work around making the start date filters work with `start_date_after` and `start_date_before`
- Improve UAT coverage around filters and sort for events
- Update event filter labels

start_date

### CODE REVIEW

File: src/apps/events/macros/event-filters-fields.js [hunk 1]
```diff
@@ -33,13 +33,13 @@ const eventFiltersFields = ({ advisers }) => {
...
       macroName: 'TextField',
-      name: 'start_date',
+      name: 'start_date_after',
       hint: 'YYYY-MM-DD',
       placeholder: `e.g. ${currentYear}-07-18`,
     },
     {
       macroName: 'TextField',
-      name: 'end_date',
+      name: 'start_date_before',
```
Comment (ztolley) [inline src/apps/events/macros/event-filters-fields.js]: is it < or <= ? Just checking you don't have to do something hacky like add or remove a day

Comment (feedmypixel) [reply src/apps/events/macros/event-filters-fields.js]: `start_date_after` includes the day you enter and any after. 
`start_date_before` includes the day you enter and any before. 
This enables the user to search for a start date between and including the dates entered.

File: test/acceptance/features/events/page-objects/EventList.js [hunk 2]
```diff
@@ -5,11 +5,34 @@ module.exports = {
...
-    firstHeaderInList: '.c-entity-list li:first-child .c-entity__header a',
-    secondHeaderInList: '.c-entity-list li:nth-child(2) .c-entity__header a',
     xhrTargetElement: '#xhr-outlet',
   },
   sections: {
+    filterTags: {
+      selector: '.c-collection__filter-summary',
+      elements: {
+        eventName: {
+          selector: '//span[contains(@class,"c-collection__filter-label")][text()[normalize-space()="Event name"]]/following-sibling::a',
```
Comment (tyom) [inline test/acceptance/features/events/page-objects/EventList.js]: @yeahfro Added a [new helper](https://github.com/uktrade/data-hub-frontend/blob/develop/test/acceptance/common/selectors.js) for selection by text. Is it something we can apply here? Something like [what we have in search](https://github.com/uktrade/data-hub-frontend/blob/develop/test/acceptance/features/search/page-objects/Search.js#L11-L19)?

Comment (feedmypixel) [reply test/acceptance/features/events/page-objects/EventList.js]: I could also make use of the [date.js](https://github.com/uktrade/data-hub-frontend/blob/develop/test/acceptance/common/date.js)

**Comment by MillsWest** [comment]
That makes sense now Ta @feedmypixel

**Pull request merged**